### PR TITLE
Remove option to build with standalone nGraph

### PR DESCRIPTION
**Pull request by adk9**
This PR removes options to build and test against standalone nGraph. The default build option now builds with nGraph distributed by OpenVINO.

### CODE REVIEW

File: python/ngraph_bridge/__init__.in.py [hunk 1]
```diff
@@ -223,5 +218,4 @@ def get_disabled_ops():
     "nGraph version used for this build: " + str(ngraph_bridge_lib.ngraph_lib_version()) + "\n" + \
```
Comment (kanvi-nervana) [inline python/ngraph_bridge/__init__.in.py:218]: would this change?

Comment (adk9) [reply python/ngraph_bridge/__init__.in.py:218]: Change to what? This is just printing out the version of nGraph in use.

Comment (kanvi-nervana) [reply python/ngraph_bridge/__init__.in.py:218]: nGraph version won't change anymore, right?
So we don't need to display it.

**Review by kanvi-nervana — COMMENTED** [review]
Remove these files as well
https://github.com/tensorflow/ngraph-bridge/blob/master/test/python/tensorflow/tests_linux_ngraph_cpu.txt
https://github.com/tensorflow/ngraph-bridge/blob/master/test/python/tensorflow/tests_linux_ngraph_interpreter.txt

File: python/ngraph_bridge/__init__.in.py (hunk 1 above)
Comment (adk9) [reply python/ngraph_bridge/__init__.in.py:218]: It's not under our control, but I agree that it isn't particularly useful to show.

Comment (kanvi-nervana) [reply python/ngraph_bridge/__init__.in.py:218]: when building with OV I have noticed that the version is 0.0.0+<some-commit-id>
eg: nGraph version used for this build: b'0.0.0+a5f5384' which is why I assumed it wouldn't change

Comment (adk9) [reply python/ngraph_bridge/__init__.in.py:218]: It identifies the git commit which changes with every version.

But if you notice carefully, our integration (build scripts, source, etc.) are still nGraph-specific; this PR only changes where the nGraph we build against comes from. Should we revisit the suggested change above as part of the OV rebranding?

Comment (kanvi-nervana) [reply python/ngraph_bridge/__init__.in.py:218]: Sure, we can do that.

**Review by bani-intelaipg — APPROVED** [review]

File: build_ngtf.py [hunk 2]
```diff
@@ -334,104 +308,54 @@ def main():
...
+
+        cmake_build(build_dir, openvino_src_dir, openvino_cmake_flags,
+                    verbosity)
 
     # Next build CMAKE options for the bridge
     ngraph_tf_cmake_flags = [
         "-DNGRAPH_TF_INSTALL_PREFIX=" + artifacts_location,
-        "-DUSE_PRE_BUILT_NGRAPH=ON",
         "-DNGRAPH_TARGET_ARCH=" + target_arch,
         "-DNGRAPH_TUNE_ARCH=" + target_arch,
```
Comment (kanvi-nervana) [inline build_ngtf.py:346]: Are these flags required for the bridge?

Comment (adk9) [reply build_ngtf.py:355]: No they're not. Good catch, I'll remove them from the build script too.

**Review by kanvi-nervana — APPROVED** [review]

File: build_ngtf.py (hunk 2 above)
Comment (adk9) [reply build_ngtf.py]: Actually, I added it to the bridge build just in case, but most importantly also pass it to OV's build (d0d573494)

**Pull request merged**